graylog2-server/src/main/java/org/graylog/security/authservice/backend/ADAuthServiceBackend.java

```diff
@@ -150,7 +150,7 @@ private Optional<LDAPUser> findUser(LDAPConnection connection, AuthServiceCreden
                 .userUniqueIdAttribute(AD_OBJECT_GUID)
                 .userNameAttribute(config.userNameAttribute())
                 .userFullNameAttribute(config.userFullNameAttribute())
-                .emailAttributes(new ArrayList<>())
+                .emailAttributes(Arrays.asList("mail"))
```

Comment:
hmm, isn't that missing the `rfc822Mailbox` attribute? looks like we've had it in the past

Reply:
I might be wrong :thinking:

Reply:
Referring to this here
https://github.com/Graylog2/graylog2-server/pull/14041/files?diff=unified&w=0#diff-dba3473472c7f22e49e817482ca7db7a3726e55cc78fb011c4c31ad64e9972d4L199-L200

Reply:
We did - is that an AD or LDAP attribute? I only saw `mail` in the AD docs.